backend/src/main/kotlin/cz/loono/backend/db/repository/AccountRepository.kt

```diff
@@ -11,4 +12,5 @@ interface AccountRepository : CrudRepository<Account, Long> {
     fun findByUid(uid: String): Account?
     fun deleteAccountByUid(uid: String): Long
     fun findAllByOrderByPointsDesc(pageable: Pageable): List<Account>
+    fun findAll(pageable: Pageable): Slice<Account>
```

Comment:
Proc Slice? Jake to ma vyhody oproti normalni kolekci? Pripadne muzeme CrudRepository zmenit na PagingAndSortingRepository.

Reply:
Slice je neco jako page, akorat nedela call, aby zjistil total count of rows. Vyhodou oproti listu je, ze nenahrajeme veskere zaznamy z DB do pometi a tim padem zbavime se OOM, predpokladam, ze accounts muze byt hodne